src/language-js/print/call-expression.js

```diff
@@ -101,12 +102,27 @@ function printCallExpression(path, options, print) {
 }
 
 function isCommonsJsOrAmdCall(node, parentNode) {
-  return (
-    node.callee.type === "Identifier" &&
-    (node.callee.name === "require" ||
-      (node.callee.name === "define" &&
-        (!parentNode || parentNode.type === "ExpressionStatement")))
-  );
+  if (node.callee.type !== "Identifier") {
+    return false;
+  }
+
+  if (node.callee.name === "require") {
+    return true;
+  }
+
+  if (node.callee.name === "define") {
+    const args = getCallArguments(node);
+    return (
+      (!parentNode || parentNode.type === "ExpressionStatement") &&
```

Comment:
I  don't think it's possible that CallExpression don't have a parent.

Reply:
I had big doubts about that too, but wasn't sure. Okay, let's remove this check.